app/views/portfolios/_earnings_summary_card.html.erb

```diff
@@ -1,25 +1,30 @@
 <div class="rounded-xl border border-black/60 overflow-hidden bg-white">
   <div class="bg-amber-300 py-3 px-5 font-medium text-sm tracking-wide rounded-t-xl">Investment Earnings Summary</div>
   <div class="divide-y divide-black/40 text-sm">
-    <% summary_rows = [
-      ['Attendance Earnings', 5.00],
-      ['Reading Earnings', 5.00],
-      ['Math Earnings', 1.00],
-      ['Rewards', 0.00]
-    ] %>
-    <% summary_rows.each do |label, amount| %>
+    <% earnings = (@earnings_summary&.earnings_breakdown || []) %>
```

Comment:
I'm thinking about the empty state 🤔 

I believe we should show this summary and every line of the summary, even if the student has no data yet. 

What do you think?